src/layer.rs

```diff
@@ -359,14 +368,36 @@ impl Layer {
     }
 
     /// Returns a mutable reference to the glyph with the given name, if it exists.
+    pub fn glyph_mut<K>(&mut self, glyph: &K) -> Option<&mut Glyph>
+    where
+        GlyphName: Borrow<K>,
+        K: Ord + ?Sized,
+    {
+        self.glyphs.get_mut(glyph).map(|g| Arc::make_mut(g))
+    }
+
+    #[doc(hidden)]
+    #[deprecated(since = "0.4.0", note = "renamed to 'glyph'")]
+    pub fn get_glyph<K>(&self, glyph: &K) -> Option<&Arc<Glyph>>
+    where
+        GlyphName: Borrow<K>,
+        K: Ord + ?Sized,
+    {
+        self.glyph(glyph)
+    }
+
+    #[doc(hidden)]
+    #[deprecated(since = "0.4.0", note = "renamed to 'glyph_mut'")]
     pub fn get_glyph_mut<K>(&mut self, glyph: &K) -> Option<&mut Glyph>
     where
         GlyphName: Borrow<K>,
         K: Ord + ?Sized,
     {
-        self.glyphs.get_mut(glyph).and_then(|g| Arc::get_mut(g))
+        self.glyph_mut(glyph)
     }
 
+    /// Returns a mutable reference to the glyph with the given name, if it exists.
+
```

Comment:
Something's missing here?

Reply:
Oops yes this was a relic of the deprecations. Nice catch!